Sprint-3/revise/implement/password-validator.test.js

```diff
@@ -14,3 +14,30 @@ To be valid, a password must:
 
 You must breakdown this problem in order to solve it. Find one test case first and get that working
 */
+
+const passwordValidator = require("./passwordValidator");
+
+
+test('Checks if password is valid', () => {
```

Comment:
This test does not check if the function can detect an invalid password that misses only one of the following conditions:
- Have at least one English uppercase letter (A-Z)
- Have at least one English lowercase letter (a-z)
- Have at least one number (0-9)
- Have at least one of the following non-alphanumeric symbols: ("!", "#", "$", "%", ".", "*", "&")

If you prepare tests to check if your function can detect each of the above cases separately, you may detect **a bug in your implementation**.